plugins/woocommerce/src/Internal/ProductFilters/CacheController.php

```
@@ -74,6 +77,29 @@ public function clear_taxonomy_hierarchy_cache( $term_id, $term_taxonomy_id, $ta
 	}
 
+	/**
+	 * Clear taxonomy hierarchy cache when term meta is updated.
+	 * This handles the case when categories are reordered (updates 'order' meta).
+	 *
+	 * @param int    $meta_id    Meta ID.
+	 * @param int    $term_id    Term ID.
+	 * @param string $meta_key   Meta key.
+	 * @param mixed  $meta_value Meta value.
+	 */
+	public function clear_taxonomy_hierarchy_cache_on_meta_update( $meta_id, $term_id, $meta_key, $meta_value ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$meta_id'. (undefined)

(UnusedFormalParameter)